block_course_contents.php

```diff
@@ -196,8 +222,16 @@ public function get_content() {
                 $enumerate = false;
             }
 
+            $sectionnumber = $i;
+
+            // If enumerating and showing section 0, then increment section number.
+            if ( ($enumerate == true) && ($enumeratesection0 == true)) {
+                $sectionnumber++;
+            }
+
             if ($enumerate) {
-                $title = html_writer::span($i, 'section-number').' '.html_writer::span($title, 'section-title');
+                $title = html_writer::span($sectionnumber, 'section-number'). ' '
+                        . html_writer::span('&nbsp;' . $title, 'section-title');
```

Comment:
What is the point of the `&nbsp;` here?

Reply:
For a little extra spacing between the number and section title.

Reply:
I am removing it and adding an extra space properly via CSS.